### PR TITLE
feat: Edit Custom Form Fields

### LINKED ISSUE #4596: Cannot remove or delete custom question
In case there is a typo an event an organizer would like to be able to edit the custom form. Also sometimes the organizer tries out this feature and just wants to be able to delete a custom form field as well.

Please provide the option to edit or delete a custom form field.
* In attendee form (Wizard step 2)
* In speaker and session form (Wizard step 5)

Screenshot from 2020-11-01 10-29-32

### PR DESCRIPTION
**Pull request by iamareebjamal**
Fixes #4596 

image

### CODE REVIEW

File: app/components/forms/wizard/custom-form-input.ts [hunk 1]
```diff
@@ -28,32 +29,45 @@ interface Args {
...
+    return this.identifier.trim().length > 0 && this.name.trim().length > 0;
   }
 
-  @action
-  addFormField(): void {
-    if (!this.validIdentifier) {
-      return;
-    }
-    this.args.customForms.pushObject(this.store.createRecord('custom-form', {
+  @computed('name', 'type')
```
Comment (niranjan94) [inline app/components/forms/wizard/custom-form-input.ts:51]: Codacy found an issue: [Missing return type on function.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6104912)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6104912&bid=19983686)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6104912&bid=19983686)

File: app/components/forms/wizard/custom-form-input.ts (hunk 1 above)
Comment (niranjan94) [inline app/components/forms/wizard/custom-form-input.ts:51]: Codacy found an issue: [Missing return type on function.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6104912)

**Comment by iamareebjamal** [comment]
Adding Editing only for now as deleting is more involved

**Pull request merged**